src/lib/template-result.ts

```diff
@@ -100,3 +100,22 @@ export class SVGTemplateResult extends TemplateResult {
     return template;
   }
 }
+
+export const templateResultsEqual =
+    (a: TemplateResult, b: TemplateResult): boolean => {
+      if (a === b) {
```

Comment:
This should be `a.strings === b.strings`.

Or better, `aStrings === bStrings` and placed after the constant definitions below.

Reply:
This is partially correct, depending on the intent of the function. If it's as you'd expect from the name, we want to check for identity first. Then you're right in that we do want to also check for `a.strings === b.strings`.

But I suspect the naming isn't quite right here...
```suggestion
      if (a === b) {
```
```suggestion
      if (a === b) {
```
```suggestion
      if (a === b) {
```
```suggestion
      if (a === b) {
```